app/routes/analytics.js

```diff
@@ -1,5 +1,5 @@
 module.exports = function(router) {
     router.get('/analytics/:campaign_id', function(request, response) {
-        response.render('analytics', {logged_in: request.user != null});
+        response.render('analytics', {user: request.user, logged_in: request.user != null});
```

Comment:
I added `request.user` to anywhere where we are rendering a server side template so that the client can access the user data. Long term if we switch to having one server side rendered template and the rest client side, this would be greatly simplified, but for now this is needed for the client to access the user.